actix-rt/Cargo.toml

```diff
@@ -23,3 +23,7 @@ futures-util = { version = "0.3.4", default-features = false, features = ["alloc
 copyless = "0.1.4"
 smallvec = "1"
 tokio = { version = "0.2.6", default-features = false, features = ["rt-core", "rt-util", "io-driver", "tcp", "uds", "udp", "time", "signal", "stream"] }
+
+[dev-dependencies]
+futures = "0.3"
```

Comment:
looks like you're only using try_join_all from this which is included in the already used `futures-util`